app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -0,0 +1,118 @@
+angular.module('containerDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.charts'])
+    .config(['$httpProvider', function($httpProvider) {
+        $httpProvider.defaults.headers.common['X-CSRF-Token'] = jQuery('meta[name=csrf-token]').attr('content');
+    }])
+    .controller('containerDashboardController', ['$scope', '$http', '$interval', "$location",
+        function($scope, $http, $interval, $location) {
+            document.getElementById("center_div").className += " miq-body";
+
+            $scope.providers = {
+                title: "Providers",
+                count: 0,
+                notifications: []
+            };
+            $scope.nodesStatus = {
```

Comment:
1. I suggest to create one object (statuses/data/whatever) and set everything as part of it, rather than having a separate variable on scope for each status.
2. Also, the names that end with "S" for plural right next to the word status make it harder to read (podSStatus)
3. Why is count hardcoded here to 0? Can't it be set dynamically later? (it is set later to the right number anyway)